NuKeeper/Commands/GitHubNuKeeperCommand.cs

```diff
@@ -32,18 +34,21 @@ internal abstract class GitHubNuKeeperCommand : CommandBase
         public bool? Consolidate { get; set; }
 
         [Option(CommandOptionType.MultipleValue, ShortName = "l", LongName = "label",
-            Description = "Label to apply to GitHub pull requests. Defaults to 'nukeeper'. Multiple labels can be provided by specifying this option multiple times.")]
+            Description =
+                "Label to apply to GitHub pull requests. Defaults to 'nukeeper'. Multiple labels can be provided by specifying this option multiple times.")]
         public List<string> Label { get; set; }
 
         [Option(CommandOptionType.SingleValue, ShortName = "g", LongName = "api",
-            Description = "GitHub Api Base Url. If you are using an internal GitHub server and not the public one, you must set it to the api url for your GitHub server.")]
+            Description =
+                "GitHub Api Base Url. If you are using an internal GitHub server and not the public one, you must set it to the api url for your GitHub server.")]
         public string GithubApiEndpoint { get; set; }
```

Comment:
We can rename the property from `GithubApiEndpoint` to something else, maybe just `ApiEndpoint` as the commandline option won't be affected.